pkg/lib/accountmanagement/redis_store.go

```diff
@@ -22,10 +22,20 @@ type RedisStore struct {
 }
 
 type GenerateTokenOptions struct {
+	// OAuth
 	UserID      string
 	Alias       string
 	MaybeState  string
 	RedirectURI string
```

Comment:
Technically speaking, UserID is shared, and only Alias, MaybeState, RedirectURI are OAuth specific.